cf-backup/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM ubuntu:14.04
+
+#ENV RS_USERNAME RS_USERNAME
+#ENV RS_API_KEY RS_API_KEY
+#ENV RS_REGION_NAME RS_REGION_NAME
+#ENV VOLUME VOLUME

```

Comment:
This env var needs to be called something different like DIRECTORY.

Volume has a very specific and well-known meaning in Docker-land. Using it here makes it very confusing.